src/client/components/vision/camera/view_model.ts

```diff
@@ -347,3 +382,16 @@ export class CameraViewModel {
     })
   }
 }
+
+function toThreeMatrix4(mat4: Matrix4Model): Matrix4 {
```

Comment:
This is in more than one place now, should it be some utility function?

Reply:
when I see it used 3 times we can move it somewhere 🙂